js/ui/components/clubhouse.js

```diff
@@ -238,7 +245,18 @@ var ClubhouseNotificationBanner = new Lang.Class({
         else
             this.actor.x = clubhouseWindowX;
 
-        this.actor.x -= this.actor.width + margin;
+        let endX = this.actor.x;
+        endX -= this.actor.width + margin;
```

Comment:
You can fold this into the previous line

Reply:
True. Will do it.